django_cache_manager/mixins.py

```diff
@@ -72,9 +68,16 @@ def invalidate_model_cache(self):
         Invalidate model cache by generating new key for the model.
         """
         logger.info('Invalidating cache for table {0}'.format(self.model._meta.db_table))
-        related_tables = set([rel.model._meta.db_table for rel in self.model._meta.get_all_related_objects()])
-        # temporary fix for m2m relations with an intermediate model, goes away after better join caching
-        related_tables |= set([field.rel.to._meta.db_table for field in self.model._meta.fields if issubclass(type(field), RelatedField)])
+        if django.VERSION >= (1, 8):
+            related_tables = set(
```

Comment:
This looks like a big change, I would keep the previous version unless something is not working